openrtb_ext/request.go

```diff
@@ -308,3 +310,576 @@ type ExtRequestPrebidDataEidPermission struct {
 	Source  string   `json:"source"`
 	Bidders []string `json:"bidders"`
 }
+
```

Comment:
I can see one wrapper file. I think it is better to keep this all together rather than either giving each object its own wrapper file or mixing in the object wrapper code with the rest of the object code (for example adding the device wrapper code to `device.go`).